reveal/run.js

```diff
@@ -35,6 +35,13 @@
       rollingLinks: true,
       slideNumber: false,
       mouseWheel: true,
+      keyboard: {
+        37: function leftArrow() { if (isRemoteMode()) Reveal.prev(); else Reveal.left(); },
+        38: function upArrow() { if (isRemoteMode()) Reveal.prev(); else Reveal.up(); },
+        39: function rightArrow() { if (isRemoteMode()) Reveal.next(); else Reveal.right(); },
+        40: function downArrow() { if (isRemoteMode()) Reveal.next(); else Reveal.down(); },
+        82: function rKey() { toggleRemoteMode(); },
+      },
```

Comment:
Maybe this could be a little less dense? What about this:

```js
37: function leftArrow() {
  if (isRemoteMode()) Reveal.prev();
  else Reveal.left();
},
```